src/DucksNet.API/Controllers/AppointmentsController.cs

```diff
@@ -31,19 +31,12 @@ public async Task<IActionResult> GetByPetID(Guid petId)
         var appointments = await _appointmentsRepository.GetAllAsync();
         return Ok(appointments.Where(a => a.PetId == petId));
     }
-
-    [HttpGet("byEmployee/{employeeId}")]
-    public async Task<IActionResult> GetByVetID(Guid vetId)
-    {
-        var appointments = await _appointmentsRepository.GetAllAsync();
-        return Ok(appointments.Where(a => a.VetId == vetId));
```

Comment:
why did you remove it?

Reply:
Because we don't support this functionality yet :) Assigning a vet to an appointment